from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_AboutWindow(object):
    def setupUi(self, AboutWindow):
        AboutWindow.setObjectName("AboutWindow")
        AboutWindow.resize(403, 250)
        AboutWindow.setMaximumSize(QtCore.QSize(403, 250))
        icon = QtGui.QIcon.fromTheme(":/appLogo/icon.png")
        AboutWindow.setWindowIcon(icon)
        AboutWindow.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.centralwidget = QtWidgets.QWidget(AboutWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.headingAbout = QtWidgets.QLabel(self.centralwidget)
        self.headingAbout.setGeometry(QtCore.QRect(170, 0, 51, 31))
        font = QtGui.QFont()
        font.setFamily("MS Sans Serif")
        font.setPointSize(12)
        font.setBold(True)
        font.setWeight(75)
        self.headingAbout.setFont(font)
        self.headingAbout.setStyleSheet("background-color: rgb(0, 170, 0);\n"
"color: rgb(0, 0, 0);")
        self.headingAbout.setObjectName("headingAbout")
        self.content = QtWidgets.QLabel(self.centralwidget)
        self.content.setGeometry(QtCore.QRect(200, 50, 181, 51))
        font = QtGui.QFont()
        font.setFamily("MS Sans Serif")
        font.setPointSize(10)
        self.content.setFont(font)
        self.content.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.content.setAlignment(QtCore.Qt.AlignLeading|QtCore.Qt.AlignLeft|QtCore.Qt.AlignTop)
        self.content.setWordWrap(True)
        self.content.setObjectName("content")
        self.versionNumber = QtWidgets.QLabel(self.centralwidget)
        self.versionNumber.setGeometry(QtCore.QRect(60, 190, 61, 20))
        self.versionNumber.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.versionNumber.setObjectName("versionNumber")
        self.nameApp = QtWidgets.QLabel(self.centralwidget)
        self.nameApp.setGeometry(QtCore.QRect(40, 30, 111, 31))
        font = QtGui.QFont()
        font.setFamily("MS Sans Serif")
        font.setPointSize(15)
        font.setBold(True)
        font.setWeight(75)
        self.nameApp.setFont(font)
        self.nameApp.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.nameApp.setObjectName("nameApp")
        self.developers = QtWidgets.QLabel(self.centralwidget)
        self.developers.setGeometry(QtCore.QRect(200, 110, 171, 41))
        self.developers.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.developers.setWordWrap(True)
        self.developers.setObjectName("developers")
        self.picture = QtWidgets.QLabel(self.centralwidget)
        self.picture.setGeometry(QtCore.QRect(40, 70, 101, 101))
        self.picture.setText("")
        self.picture.setPixmap(QtGui.QPixmap(":/appLogo/icon.png"))
        self.picture.setScaledContents(True)
        self.picture.setObjectName("picture")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(250, 160, 51, 31))
        self.label.setText("")
        self.label.setPixmap(QtGui.QPixmap(":/comLogo/thunder_logo.png"))
        self.label.setScaledContents(True)
        self.label.setObjectName("label")
        AboutWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(AboutWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 403, 21))
        self.menubar.setObjectName("menubar")
        AboutWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(AboutWindow)
        self.statusbar.setObjectName("statusbar")
        AboutWindow.setStatusBar(self.statusbar)

        self.retranslateUi(AboutWindow)
        QtCore.QMetaObject.connectSlotsByName(AboutWindow)

    def retranslateUi(self, AboutWindow):
        _translate = QtCore.QCoreApplication.translate
        AboutWindow.setWindowTitle(_translate("AboutWindow", "About"))
        self.headingAbout.setText(_translate("AboutWindow", "About"))
        self.content.setText(_translate("AboutWindow", "PyQRCode is a Python application for Generating and Reading QRCode."))
        self.versionNumber.setText(_translate("AboutWindow", "Version: 1.1"))
        self.nameApp.setText(_translate("AboutWindow", "PyQRCode"))
        self.developers.setText(_translate("AboutWindow", "Developed by Thunder"))
import logo_rc

